docs/features.md

```diff
@@ -17,7 +17,7 @@ You can watch demos for some of these features [below](#demos).
 - [Module name suggestions](#module-names) for insertion or correction
 - [Call hierarchy support](#call-hierarchy)
 - [Qualify names from an import declaration](#qualify-imported-names) in your code
-- [Suggest alternate numeric formats](#alternate-number-formatting)
+- [Suggest alternate numeric formats](#alternate-number-formatting). This plugin is not included by default yet.
```

Comment:
Maybe it's worth to mention the issue as a reason of non-defaulting?

Reply:
Good idea, done